test/unit/1__start.js

```diff
@@ -0,0 +1,5 @@
+import Renderer from './mock';
+
+// to add global.document
```

Comment:
Will this make my problem in #1303 away ?

Reply:
yes